app/reducers/users.js

```diff
@@ -156,7 +156,15 @@ export function targetDevices(state = {}, action) {
       let newState = {};
       _.each(memberships, (membership) => {
         if (isPwd(membership)) {
-          newState[membership.userid] = _.get(membership, ['profile', 'patient', 'targetDevices'], []);
+          let targetDevices = _.get(membership, ['profile', 'patient', 'targetDevices'], []);
+          // collapse all bayercontour devices into bayercontournext
+          targetDevices = _.uniq(_.map(targetDevices, function(device) {
+            if (device.startsWith('bayercontour')) {
```

Comment:
While this will work for now, there will be a separate driver (and entry) for the Bayer Contour meter, see https://github.com/tidepool-org/chrome-uploader/pull/168/files. Probably better to match on `bayercontournextusb`, `bayercontourusb`, `bayercontournextlink` and `bayercontournext`. We'll also need to merge `abbottfreestylefreedomlite` into `abbottfreestylelite`.